dist/sweetalert2.common.js

```diff
@@ -628,29 +644,36 @@ var setParameters = function setParameters(params) {
   // Edit text on cancel and confirm buttons
   confirmButton.innerHTML = params.confirmButtonText;
   cancelButton.innerHTML = params.cancelButtonText;
+  altActionButton.innerHTML = params.altActionButtonText;
 
   // Set buttons to selected background colors
   if (params.buttonsStyling) {
     confirmButton.style.backgroundColor = params.confirmButtonColor;
     cancelButton.style.backgroundColor = params.cancelButtonColor;
+    altActionButton.style.backgroundColor = params.confirmButtonColor;
```

Comment:
I think this way the altActionButton's style should be equal to params.altActionButtonColor.